src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritServer/index.jelly

```diff
@@ -121,386 +132,311 @@
-                                 help="/plugin/gerrit-trigger/help-GerritVerifiedCmdBuildNotBuilt.html">
-                            <f:textarea name="gerritVerifiedCmdBuildNotBuilt"
-                                       value="${it.config.gerritCmdBuildNotBuilt}"
-                                       default="gerrit review &lt;CHANGE&gt;,&lt;PATCHSET&gt; --message 'No Builds Executed &lt;BUILDS_STATS&gt;' --verified &lt;VERIFIED&gt; --code-review &lt;CODE_REVIEW&gt;"/>
-                        </f:entry>
-                        <f:entry title="${%Aborted}"
-                                 help="/plugin/gerrit-trigger/help-GerritVerifiedCmdBuildAborted.html">
-                            <f:textarea name="gerritVerifiedCmdBuildAborted"
-                                        value="${it.config.gerritCmdBuildAborted}"
-                                        default="gerrit review &lt;CHANGE&gt;,&lt;PATCHSET&gt; --message 'Build Aborted &lt;BUILDS_STATS&gt;' --verified &lt;VERIFIED&gt; --code-review &lt;CODE_REVIEW&gt;"/>
-                        </f:entry>
-                    </f:section>
-                    <f:section title="${%Miscellaneous}">
-                        <f:entry title="${%Build Schedule Delay}"
-                                 help="/plugin/gerrit-trigger/help-BuildScheduleDelay.html">
-                            <f:textbox name="buildScheduleDelay"
-                                       value="${it.config.buildScheduleDelay}"
-                                       default="${com.sonyericsson.hudson.plugins.gerrit.gerritevents.GerritDefaultValues.DEFAULT_BUILD_SCHEDULE_DELAY}"
-                                       checkUrl="'${rootURL}/${serverURL}/nonNegativeIntegerCheck?value='+escape(this.value)"/>
+                <f:section title="${%Gerrit Verified Commands}" icon="symbol-terminal-outline plugin-ionicons-api">
+                    <f:entry title="${%Started}"
+                             help="/plugin/gerrit-trigger/help-GerritVerifiedCmdBuildStarted.html">
+                        <f:textarea name="gerritVerifiedCmdBuildStarted"
+                                    value="${it.config.gerritCmdBuildStarted}"
+                                    default="gerrit review &lt;CHANGE&gt;,&lt;PATCHSET&gt; --message 'Build Started &lt;BUILDURL&gt; &lt;STARTED_STATS&gt;' --verified &lt;VERIFIED&gt; --code-review &lt;CODE_REVIEW&gt;"/>
+                    </f:entry>
+                    <f:entry title="${%Successful}"
+                             help="/plugin/gerrit-trigger/help-GerritVerifiedCmdBuildSuccessful.html">
+                        <f:textarea name="gerritVerifiedCmdBuildSuccessful"
+                                    value="${it.config.gerritCmdBuildSuccessful}"
+                                    default="gerrit review &lt;CHANGE&gt;,&lt;PATCHSET&gt; --message 'Build Successful &lt;BUILDS_STATS&gt;' --verified &lt;VERIFIED&gt; --code-review &lt;CODE_REVIEW&gt;"/>
```

Comment:
I just merged a PR that added a parameter here to the defaults, so I guess there will be a merge conflict that needs to be resolved.

Reply:
Or maybe I didn't :thinking: